docker-examples/docker-compose-postgres.yaml

```diff
@@ -88,7 +90,7 @@ services:
       - GOLEM__COMPILED_TEMPLATE_SERVICE__TYPE="Local"
       - GOLEM__SHARD_MANAGER_SERVICE__TYPE="Grpc"
     ports:
-      - "${WORKER_EXECUTOR_HTTP_PORT}:${WORKER_EXECUTOR_HTTP_PORT}"
+      - "${WORKER_EXECUTOR_HTTP_PORT:-8082}:${WORKER_EXECUTOR_HTTP_PORT:-8082}"
```

Comment:
What is the difference of this with having the dafult values as .env variable?

Reply:
I am not into this style, of default values. Default values already exist in .env file.

Pros: They can download docker-compose file
cons: Prone to more user errors, because variable values are repeated. The sole reason for why I avoided repetitions, was, when I was writing compose file, I made errors with respect to port numbers

Reply:
as i mentioned in different comment, with current redis setup, internal redis port will be 6379, REDIS_PORT will change just public value